components/atoms/colors.ts

```diff
@@ -0,0 +1,15 @@
+export const Black100 = '#08111D'
```

Comment:
These colors are too ambiguous for me, but i saw that they are named like this in figma too :(

Reply:
unfortunate :( if we had something like light/dark/default we could create an object here and add the colors like
blue.light / blue.dark ...

this 100 / 010 is more prone to errors, but nevermind, better keep it consistent